ui/raidboss/data/03-hw/raid/a5s.txt

```diff
@@ -10,9 +10,8 @@ hideall "Relaxant"
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
 
 ### Hummel and Friends
-1000.0 "--sync--" sync / 00:0839::The Clevering will be sealed off/ window 1000,0
-1001.8 "--sync--" sync / 1[56]:[^:]*:Faust:367:/ window 1002,0.5
-1009.8 "Kaltstrahl x1" sync / 1[56]:[^:]*:Faust:16CC:/
+1000.0 "--sync--" sync / 104:[^:]*:1($|:)/ window 1000,1
```

Comment:
I think it'd be better to leave the zone seal here, rather than incorrectly starting at t=1000 for the second half of the fight.  This is the same as a9s, imo.